xknx/dpt/dpt_7.py

```diff
@@ -24,19 +24,19 @@ class DPT2ByteUnsigned(DPTNumeric):
 
     value_min = 0
     value_max = 65535
-    resolution = 1
+    resolution: float = 1
```

Comment:
Yes. This is an artifact of a previous implementation of mine using fractional resolution and seconds. I initially added the coercion to float because the linter complained about the type (while testing was fine).

I will revert this change.